samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -79,6 +79,18 @@ Recovery mode
 In the provisioned state, you can enter the recovery mode for a limited time.
 This mode allows the Android device to recover a lost provisioning key from the locator tag.
 
+DULT Motion detector active mode
+--------------------------------
+
+In the provisioned state, the accessory can enter the DULT motion detector active mode.
+It happens when the accessory is separated from the owner for a randomized amount of time between :kconfig:option:`CONFIG_DULT_MOTION_DETECTOR_SEPARATED_UT_TIMEOUT_PERIOD_MIN` and :kconfig:option:`CONFIG_DULT_MOTION_DETECTOR_SEPARATED_UT_TIMEOUT_PERIOD_MAX`.
+In this state, if the motion is detected, the accessory starts the ringing action.
+It is done to inform the non-owner that is moving the accessory, that the device might have been used to track the non-owner.
+On DKs the simulated motion can be generated by button action.
+On Thingy:53 the built-in accelerometer is used to detect the motion.
+The motion detector is deactivated for :kconfig:option:`CONFIG_DULT_MOTION_DETECTOR_SEPARATED_UT_BACKOFF_PERIOD` after the :kconfig:option:`CONFIG_DULT_MOTION_DETECTOR_SEPARATED_UT_MAX_SOUND_COUNT` number of sounds has been played or after the motion has been detected and :kconfig:option:`CONFIG_DULT_MOTION_DETECTOR_SEPARATED_UT_ACTIVE_POLL_DURATION` amount of time has passed.
```

Comment:
I will remove the max count and poll duration, but I'd like to keep backoff kconfig, because later on I stress the importance of enabling the test mode to shorten the periods.